framework/lib/components/tag/tag.tsx

```diff
@@ -7,18 +7,50 @@ import { TagProps } from './types'
  * @see {@link https://northlight.dev/reference/tag}
  *
  * @example
- * (? <Tag>Brand 1</Tag> ?)
+ * (? <Tag>Hello</Tag> ?)
  *
  * @example
  * (?
- *   <Tag>
-    <TagLeftIcon>
-      <Icon as={ StackSolid } />
-    </TagLeftIcon>
-    <TagLabel>Left Icon</TagLabel>
-  </Tag>
+ * +
+ * const colors = ["mediatoolBlue", "blue", "gray", "red", "green",
+ * "orange", "yellow", "teal", "purple", "pink"]
+ * const sizes = ["sm", "md", "lg"]
```

Comment:
I will let you play around with this one. Based on the [design requierements](https://www.figma.com/file/0rnkmzpBhBoVl3PsbZDmGD/%E2%AC%9B%EF%B8%8F%C2%A0%C2%A0Core.mainCOMPONENTS?type=design&node-id=2-28&mode=design)

The `size="sm"` tag uses icon `size="xs"`
BUT
The `size="md"` tag uses icon `size="md"`
and The` size="lg"` tag uses icon `size="lg"`

Let's show this properly in the example. This is very important for the **Developer Experience** because the Developers will adopt the way that is served in the documentation, and these finesses or aesthetic models will be just copied/pasted in the next use of the tag.